doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_dfu.rst

```diff
@@ -337,35 +345,35 @@ The examples below demonstrate the use of these variables and methods.
 Set component definition and memory ranges
 ------------------------------------------
 
-In the :file:`app_envelope_yaml.jinja2` (found `here <https://github.com/nrfconnect/suit-generator/blob/main/ncs/app_envelope.yaml.jinja2>`__), the component definition and memory ranges are filled out by using the ``edtlib`` object like so:
+In the :file:`app_envelope_yaml.jinja2` (found `here <https://github.com/nrfconnect/suit-generator/blob/v2.7.0/ncs/app_envelope.yaml.jinja2>`__), the component definition and memory ranges are filled out by using the ``edtlib`` object like so:
```

Comment:
What is `edtlib` ? It isn't explained anywher, I do not understand what it is at nor will the user. Shouldn't we delete this information?

Reply:
Earlier in the same doc, we mention this: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_dfu.html#variables_and_methods_available_in_the_manifest_templates
And provide a link to [edtlib](https://python-devicetree.readthedocs.io/en/latest/edtlib.html).

I think this is a valid point you've raised - if it is still unclear what ``edtib`` is, then we can add more info to the section where I first linked to, perhaps?

Reply:
Ok, I think it's ok if i just add the comment in the bracket that this is about devicetree values, then it is ok for me.